Description: require-securerandom.
securerandom is a inbuilt library in ruby. Calling it explicitly to the scope. 
Thanks to Shanavas <shanavas@disroot.org>. 
Author: Abhijith PA <abhijith@openmailbox.org>
Forwarded: not-needed
Last-Update: 2017-06-02


Index: ruby-secure-headers/spec/spec_helper.rb
===================================================================
--- ruby-secure-headers.orig/spec/spec_helper.rb
+++ ruby-secure-headers/spec/spec_helper.rb
@@ -5,6 +5,7 @@ require 'pry-nav'
 
 require 'coveralls'
 Coveralls.wear!
+require 'securerandom'
 
 require File.join(File.dirname(__FILE__), '..', 'lib', 'secure_headers')
 
